ui/src/router/components/configuration/components/ExperimentConfigSection.js

```diff
@@ -25,6 +27,9 @@ const FallbackView = ({ text }) => (
 );
 
 const CustomExperimentConfigView = ({ projectId, remoteUi, config }) => {
+  const [configReady, setConfigReady] = useState(false);
+  const [configFailed, setConfigFailed] = useState(false);
+
   // Retrieve script from host dynamically
   const { ready, failed } = useDynamicScript({
```

Comment:
Shall we replace this with `LoadDynamicScript` too instead of directly calling the hook?

Also, since we have similar logic in multiple places now, there is likely some scope for refactoring this further to reduce some code duplication by wrapping all the loading into a single component.